### DISCUSSION TITLE
Support to export logs more than 10000 rows per time and view more than 500 rows per page.

### DISCUSSION THREAD

**Discussion by Thaiph1308** [openobserve/openobserve#3701] [Q&A] [answered]
Is there any way to download logs in batches of more than 10,000 rows at a time? I have millions of lines of logs that need to be downloaded. Additionally, is it possible to view more than 500 rows per page?

**Comment by prabhatsharma** [comment, marked as answer]
You can download the parquet files directly that have log data if you are self hosting OpenObserve to download million of lines. You should be able to point tools like AWS glue and amazon athena to openobserve log files in s3 and be able to query and extract any amount of data if that is what you are looking for. Other standard tools like spark and duckdb should also work.
